pyproject.toml

```diff
@@ -54,3 +58,6 @@ packages = [
 
 [tool.setuptools.package-data]
 "*" = ["*.yaml", "*.yml", "*.json"]
+
+[tool.pytest.ini_options]
+testpaths = ["scripts/generate_readme/tests"]
```

Comment:
I think we need a generic way to handle tests, like we've done in https://github.com/kubeflow/pipelines-components/tree/main/.github/scripts. But we can do that later.